### PR TITLE
Add "Donate" section

### LINKED ISSUE #14: [FEATURE]: Add donation button/widget
It would be nice if we could add a donation button somewhere on the page. People will be able to find it easier now. 🙂

[Donate](https://liberapay.com/TheAlgorithms/donate)

We can also add a widget of how much money the organization is receiving per week:

[Income](https://liberapay.com/TheAlgorithms)

What do you think?

### PR DESCRIPTION
**Pull request by maltejur**
Closes #14

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -144,6 +144,34 @@ export default function Home({ topAlgorithms, featuredAlgorithms }) {
                     please fork the repository, work on your changes and then
                     submit them as a pull request.
                   </Typography>
+                  <Typography variant="h5" className={classes.title}>
+                    Donate
+                  </Typography>
+                  <Typography>
+                    Another way you can support us is to make a donation via
+                    Liberapay. Even a small donation is much appreciated.
```
Comment (maltejur) [inline pages/index.tsx]: Can you help to expand this text some more?

Comment (Panquesito7) [inline pages/index.tsx]: Something like this?
```suggestion
                    Liberapay. Even a small donation is much appreciated.
                    By donating, it means that you appreciate and like our work. If you don't like our work,
                    there's no need to donate. If you donate, top members will be able to contribute further to
                    The Algorithms projects. We appreciate donations from everyone,
                    from everywhere, no matter the amount.
```

Comment (Panquesito7) [inline pages/index.tsx]: You can maybe here add a link to [Liberapay](https://liberapay.com) official website.

Comment (maltejur) [reply pages/index.tsx]: 47dc26c6dc8fa75eb26479fc22b7438b1c602935

Comment (maltejur) [reply pages/index.tsx]: 47dc26c6dc8fa75eb26479fc22b7438b1c602935

**Review by Panquesito7 — APPROVED** [review]
LGTM. Thank you! 👍 🎉

**Pull request merged**